tools/DfmHttpService/Utility.cs

```diff
@@ -56,12 +56,12 @@ public static CommandMessage GetCommandMessage(HttpListenerContext context)
             return message;
         }
 
-        public static void ReplySuccessfulResponse(HttpListenerContext context, string content, string contentType)
```

Comment:
But matbe someone else will use that in the future, so keep it

---
In reply to: [111541827](https://github.com/dotnet/docfx/pull/1547#discussion_r111541827) [](ancestors = 111541827,111536407)